src/topology/instances/rat.lean

```diff
@@ -0,0 +1,88 @@
+/-
+Copyright (c) 2022 Yury Kudryashov. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Yury Kudryashov
+-/
+import topology.instances.irrational
+import topology.alexandroff
+
+/-!
+# Topology on rational numbers
+
+The structure of a metric space on `ℚ` is introduced elsewhere, induced from `ℝ`. In this file we
+prove some properties of this topological space and its one-point compactification.
+
+## Main statements
+
+- `rat.totally_disconnected_space`: `ℚ` is a totally disconnected space;
+
+- `rat.not_countably_generated_nhds_infty_alexandroff`: the filter of neighbourhoods of infinity in
+  `alexandroff ℚ` is not countably generated.
+
+## Notation
+
+- `ℚ∞` is used as a local notation for `alexandroff ℚ`
+-/
+
+open set metric filter topological_space
+open_locale topological_space alexandroff
+local notation `ℚ∞` := alexandroff ℚ
```

Comment:
Should this be added to the `alexandroff` locale, or do you think that's not helpful?

Reply:
I don't think that we'll use `alexandroff ℚ` for any purpose other than "build a strange counterexample".